context/opencensus-context/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.2.dev0'
+__version__ = '0.1.0'
```

Comment:
Should this be `0.2.0` or `0.1.1`?

Reply:
Current version is 0.1.0, and there is no change, so we don't need a new release.